lib/clients/client.ex

```diff
@@ -28,6 +28,7 @@ defmodule GlassFactoryApi.Clients.Client do
         office_id: 789
       }
   """
+  @spec to_struct(map()) :: Client.t()
```

Comment:
When the type is defined in the same module as the function we usually don't namespace it, e.g. https://github.com/elixir-lang/elixir/blob/v1.9.4/lib/elixir/lib/string.ex#L787

```suggestion
  @spec to_struct(map) :: t
```